event.go

```diff
@@ -33,7 +33,7 @@ type EventRequest struct {
 
 // EventData is the unmarshalled object as a map.
 type EventData struct {
-	Object             map[string]interface{}
+	Object             map[string]interface{} `json:"-"`
```

Comment:
Do you think that we could add a comment here that explains that despite the `json:"-"`, this property will be populated independently? Maybe something like:

> Object is a raw mapping of the API resource contained in the event.
>
> Although marked with `json:"-"`, it's still populated independently by a custom `UnmarshalJSON` implementation.